scripts/index.js

```diff
@@ -1,12 +1,38 @@
 
 // Implement the following functions to fulfill the tests!
-function getLocationName() {}
-function getLocationCountry() {}
-function getLocationLatitude() {}
-function getLocationLongitude() {}
-function getDescription() {}
-function getWindSpeed() {}
-function getSunrise() {}
+function getLocationName(location) {
+    location = location.name;
```

Comment:
Nicely done, but you might not need to reassign the argument to a new value. Not an issue for these small examples, but consider making a new variable instead.